eli5/xgboost.py

```diff
@@ -97,7 +99,10 @@ def explain_prediction_xgboost(
         target_names=None,
         targets=None,
         feature_names=None,
-        vectorized=False):
+        vectorized=False,
+        feature_re=None,
+        feature_filter=None,
```

Comment:
Right, I like that feature_re and feature_filter come after feature_names :) Fixed it here too in 724c63a